.github/workflows/ci.yaml

```diff
@@ -172,7 +173,7 @@ jobs:
       - name: Set DEV release version
         run: |
           poetry install --with build -n
-          poetry version $(poetry run semantic-release version --print)-dev.${{ github.run_id }}
+          poetry version $(poetry run semantic-release --strict --verbose version --print)-dev.${{ github.run_id }}
```

Comment:
Required to get the full barnch history so that semantic-release can calc the version